### PR TITLE
Downlevel Omit when used as ExpressionWithTypeArguments

### LINKED ISSUE #33: `Omit` not replaced in `extends`
```ts
import { RawSourceMap } from 'source-map';

interface FixedRawSourceMap extends Omit<RawSourceMap, 'version'> {
    version: number;
}
```

Output:

```ts
import { RawSourceMap } from 'source-map';
interface FixedRawSourceMap extends Omit<RawSourceMap, 'version'> {
    version: number;
}
```

Using 0.4.0

Workaround:

```ts
import { RawSourceMap } from 'source-map';

type SourceMapWithVersion = Omit<RawSourceMap, 'version'>;

interface FixedRawSourceMap extends SourceMapWithVersion {
  version: number;
}
```

### PR DESCRIPTION
**Pull request by chrisguttandin**
This pull request is intended to fix #33 

The code is very similar to the one which down-levels Omit when used in a TypeReferenceNode. Should I refactor it into a function which can be used for both cases?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -123,6 +123,28 @@ function doTransform(checker, k) {
...
+      ) {
+        return ts.createTypeReferenceNode(ts.createIdentifier("Pick"), [
+          typeArguments[0],
+          ts.createTypeReferenceNode(ts.createIdentifier("Exclude"), [
+            ts.createTypeOperatorNode(typeArguments[0]),
+            typeArguments[1]
+          ])
+        ]);
+      }
     } else if (ts.isTypeReferenceNode(n) && ts.isIdentifier(n.typeName) && n.typeName.escapedText === "Omit") {
```
Comment (sandersn) [inline index.js:148]:
```suggestion
    } else if (ts.isTypeReferenceNode(n) && ts.isIdentifier(n.typeName) && n.typeName.escapedText === "Omit"
      || ts.isExpressionWithTypeArguments(n) && ts.isIdentifier(n.expression) && n.expression.escapedText === "Omit") {
```

File: index.js [hunk 2]
```diff
@@ -123,6 +123,28 @@ function doTransform(checker, k) {
...
       const symbol = checker.getSymbolAtLocation(n.typeName);
```
Comment (sandersn) [inline index.js:149]:
```suggestion
      const symbol = checker.getSymbolAtLocation(ts.isTypeReferenceNode(n) ? n.typeName : n.expression);
```

**Comment by sandersn** [comment]
Also, sorry for the long delay -- I only look at downlevel-dts during the RC period for each Typescript release. Let me know if you'd like me to take over the PR if you've timed out.

**Comment by chrisguttandin** [comment]
No worries. I'm afraid I don't have the time to address your feedback this week. If you have the capacity to take this over I would be grateful.

**Pull request merged**